.ci/azure-pipelines-main.yml

```diff
@@ -55,6 +55,13 @@ jobs:
           packageType: sdk
           version: ${{ parameters.DotNetSdkVersion }}
 
+      - task: SonarCloudPrepare@1
+        displayName: 'Prepare analysis on SonarCloud'
+        inputs:
+          SonarCloud: SC
```

Comment:
@EraYaN This will need to be updated, I'm not exactly sure what it should be but I just put this `SC` value in as a placeholder